mne_bids_pipeline/_config_utils.py

```diff
@@ -159,9 +172,9 @@ def get_subjects_sessions(config: SimpleNamespace) -> dict[str, list[str] | list
                 f"{tuple(sorted(missing_sessions))}, and "
                 "`config.allow_missing_sessions` is False"
             )
-        subj_sessions[subject] = sorted(set(cfg_sessions) & set(valid_sessions_subj))
-        if subj_sessions[subject] == []:
-            subj_sessions[subject] = [None]
```

Comment:
this wasn't doing what was intended: the `None` was getting interpreted as "all sessions" 🤦🏻 Now, if a subject doesn't have data for any sessions, we just omit their entry from this `subj_sessions` dict.